src/App.css

```diff
@@ -1,12 +1,86 @@
-#nav-brand-text {
-  color: white;
-}
 
-#total-losses-value {
-  color: red;
+.main-container {
+  font-family: var(--content-font), Arial, Helvetica, sans-serif;
 }
 
 #losses-over-time-card {
   width: 100%;
   height: 40vh;
 }
+
+.row {
+display: flex;
+align-items: center;
+}
+
+.row:not(:last-child) {
+  margin-bottom: 15px;
+}
+
+h1 {
+  font-family: var(--title-font);
```

Comment:
My CSS skills have somewhat decayed so I don't even know what a lot of these rules even do.